docs/docs/learn/shards/primer/coding-with-shards.md

```diff
@@ -445,6 +445,19 @@ When the Mesh is run, the Wires are executed in sequence and your program is sta
         @run(mesh-name (1.0 | Math.Divide(60.0)))
         ```
 
+!!! note "Parentheses to Control Evaluation Order"
```

Comment:
This is important, we probably want a section above, rather than just a note as it's a language construct